spec/bolt/cli_spec.rb

```diff
@@ -931,7 +931,6 @@ def stub_config(file_content = {})
           json = JSON.parse(output.string)
           expect(json).to eq(
             [
-              ["apply::resource", "Apply a single Puppet resource"],
```

Comment:
Can we make this an include rather than an eq?  That wont' solve this problem but will make it easier to add new content. I don't think the point of this test is to make sure we have bundled the things we're supposed to have bundled because it doesn't execute at the right point in the pipeline